src/test-collection/index.ts

```diff
@@ -172,4 +206,22 @@ export class TestCollection {
 
         return this;
     }
+
+    format(formatterType: ValueOf<typeof Formatters>): (FormatterListTest | FormatterTreeMainRunnable)[] {
```

Comment:
Here I see no benefit in using formatters via TestCollection as opposed to using them directly. IMO it only introduces additional logic, constants, checks, etc. I still think it would be nice to get rid of the whole TestCollection class in the future =)

Not critical, up to you to decide